src/redux/auth/saga.ts

```diff
@@ -192,8 +196,8 @@ export function* resetPasswordSaga({
 }: ResetPasswordAction) {
   try {
     const actionCodeSettings = {
-      // to control if back to app after password change in browser
-      // url: `https://www.example.com/?email=${email}`,
+      // TODO: check if it's required
+      url: '',
```

Comment:
@bastej , I had to uncomment this property to make TS not complain about types (later, line 204-206). I'm not sure about your implementation - is it okay? What should be this `url`?

Reply:
We will be able to generate url to redirect after we build standalone app version